packages/contracts-bedrock/contracts/test/FaultDisputeGame.t.sol

```diff
@@ -439,14 +446,24 @@ contract GamePlayer {
         return uint256(vm.load(address(gameProxy), bytes32(uint256(1))));
     }
 
-    /// @notice Returns the player's claim that commits to a given gindex.
-    function claimAt(Position _position) internal view returns (Claim claim_) {
-        return claimAt(_position.rightIndex(maxDepth).indexAtDepth());
+    /// @notice Returns the preimage of a player's claim that commits to a given trace index.
+    function traceAt(Position _position) public view returns (bytes memory trace_) {
+        return traceAt(_position.rightIndex(maxDepth).indexAtDepth());
```

Comment:
Could offload the `_position.rightIndex(maxDepth).indexAtDepth()` chained call to `LibPosition` with
`function traceIndex(Position _position, uint256 _maxDepth) internal pure returns (uint64 indexAtDepth_)`

And then chain the calls inside `traceIndex` function.
So this would end up just being:
```suggestion
        return traceAt(_position.traceIndex(maxDepth));
```